SparkPost/2web2ui
# SA-1541: Pulls subaccounts on first mount

### What Changed
 - Subaccounts pulled down on mounting report builder table
 - Subaccounts no longer show "Deleted" if it can't find the name

### How To Test
 - Go to /signals/analytics (Hibana)
 - Verify subaccounts endpoint is hit
 - Use `Group By` select and choose subaccounts at bottom of table
 - Verify that the table no longer shows `Deleted` for subaccounts with missing names
